tst/EnergyPlus/unit/EPVector.unit.cc

```diff
@@ -71,12 +72,44 @@ TEST_F(EnergyPlusFixture, EPVectorTest_Basic)
     ASSERT_THROW(v[4] = 5, std::out_of_range);
     ASSERT_THROW(v(5) = 5, std::out_of_range);
 #endif // !NDEBUG
+    v.resize(2);
+    EXPECT_EQ(2, v.size());
+    EXPECT_EQ(1, v(1));
+    EXPECT_EQ(2, v(2));
+    v.resize(4);
+    EXPECT_EQ(1, v(1));
+    EXPECT_EQ(2, v(2));
+    EXPECT_EQ(0, v(3));
+    EXPECT_EQ(0, v(4));
+    v.clear();
+    EXPECT_FALSE(v.allocated());
```

Comment:
In the original implementation, this will fail because `clear` didn't change the `m_allocated` member variable.